### PR TITLE
REVAI-3853: usage-examples

### PR DESCRIPTION
**Pull request by kirillatrev**
REVAI-3853: usage-examples

### CODE REVIEW

**Review by amikofalvy — APPROVED** [review]

File: examples/async_translate_local_media_file.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        if (err) throw err;
+        console.log('Success! Check the examples/outputs/ directory for the transcript.')
+    });
+
+    /**
+     * Delete a job
+     * Job deletion will remove all information about the job from the servers
+     */
+    // await client.deleteJob(job.id);
+})();
```
Comment (alexsku) [inline examples/async_translate_local_media_file.js:78]: why is this connected out? if we don’t need it i would remove it and if we need-uncomment

**Review by alexsku — APPROVED** [review]
:shipit:

File: examples/async_summarize_local_media_file.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+const { SummarizationJobStatus } = require('../dist/src/models/async/SummarizationJobStatus');
+const { SummarizationFormattingOptions } = require('../dist/src/models/async/SummarizationFormattingOptions');
+const token = require('./config/config.json').access_token;
+
+(async () => {
+    // Initialize your client with your Rev AI access token
+    var client = new revai.RevAiApiClient({ token: token });
+
+    // Get account details
+    var account = await client.getAccount();
```
Comment (dmtrrk) [inline examples/async_summarize_local_media_file.js]:
```suggestion
    const account = await client.getAccount();
```

File: examples/async_summarize_local_media_file.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    console.log(`Summarization status: ${job.summarization.status}`);
+    console.log(`Created On: ${job.created_on}`);
+
+    /**
+     * Waits 5 seconds between each status check to see if job is complete.
+     * note: polling for job status is not recommended in a non-testing environment.
+     * Use the notification_config option (see: https://docs.rev.ai/sdk/node/)
+     * to receive the response asynchronously on job completion
+     */
+    while((job = await client.getJobDetails(job.id))  && (
```
Comment (dmtrrk) [inline examples/async_summarize_local_media_file.js]:
```suggestion
    while((job = await client.getJobDetails(job.id)) && (
```

File: examples/async_summarize_local_media_file.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        ))
+    {
+        console.log(`Job ${job.id} is ${job.status}, summarization is ${job.summarization.status}`);
+        await new Promise( resolve => setTimeout(resolve, 5000));
+    }
+
+    /**
+     * Get transcript summary as plain text
+     */
+    var transcriptSummary = await client.getTranscriptSummaryText(job.id);
```
Comment (dmtrrk) [inline examples/async_summarize_local_media_file.js]:
```suggestion
    const transcriptSummary = await client.getTranscriptSummaryText(job.id);
```

File: examples/async_translate_local_media_file.js (hunk 1 above)
Comment (kirillatrev) [reply examples/async_translate_local_media_file.js:78]: Some of the code in the existing example files is there just as an example and is commented out.

**Review by dmtrrk — APPROVED** [review]
Please check var vs const, indentation
Also, check for variable hoisting cases, eg `var job`. I think it is redundant

**Comment by kirillatrev** [comment]
@dmtrrk hoisting - I think all examples follow the same pattern. Also I believe this is standard in JS, unless I totally missed your point....

**Pull request merged**